devp2p/kademlia.py

```diff
@@ -220,6 +220,19 @@ def split_bucket(self, bucket):
         self.buckets[index] = a
         self.buckets.insert(index + 1, b)
 
+    def getCurrentNetSizeEst(self):
```

Comment:
This project is using snake_case names for functions instead of camelCase names.